src/app/credExplorer/App.js

```diff
@@ -61,14 +65,11 @@ export function createApp(
       const {localStore} = this.props;
       const {appState} = this.state;
       const loadingState =
-        appState.type === "INITIALIZED" ? appState.substate.loading : null;
+        appState.type === "UNINITIALIZED" ? null : appState.loading;
```

Comment:
The nullity of this variable makes it extra confusing to clients: a
client who writes `loadingState === "LOADING"` will be unintentionally
forgetting the `null` case, which is not locally apparent.

In other words, clients should expect that `loadingState: LoadingState`,
but this constraint does not in fact hold.

As this variable is only used once, and the caller has a redundant check
against the app state anyway, can we inline it?